app/src/contexts/CurrentGame.ts

```diff
@@ -9,5 +9,6 @@ export const CurrentGameContext = createContext<CurrentGameContextType>({
   rounds: [],
   cards: [],
   players: [],
-  turns: []
+  turns: [],
+  allow_card_skips: false
```

Comment:
```suggestion
  allow_card_skips: true
```

as the default. im not really sure when these context values ever get used tho. always comes from first definition of provider?

Reply:
oh nice - thanks for linking to docs - no difference then in prod